### DISCUSSION TITLE
error - ESLint: Cannot find module './common'  at VERCEL

### DISCUSSION THREAD

**Discussion by guimteixeira** [vercel/next.js#39850] [Help]
When I Deploy a Next project to VERCEL I get this error:

error - ESLint: Cannot find module './common' Require stack: - /vercel/path1/node_modules/debug/src/node.js - /vercel/path1/node_modules/debug/src/index.js - /vercel/path1/node_modules/@eslint/eslintrc/dist/eslintrc.cjs - /vercel/path1/node_modules/eslint/lib/cli-engine/cli-engine.js - /vercel/path1/node_modules/eslint/lib/eslint/eslint.js - /vercel/path1/node_modules/eslint/lib/eslint/index.js - /vercel/path1/node_modules/eslint/lib/api.js - /vercel/path1/node_modules/next/dist/lib/eslint/runLintCheck.js - /vercel/path1/node_modules/next/dist/compiled/jest-worker/processChild.js

Here is my package.json

`{
  "name": "mdb-picodogaviao",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "mongoose": "^6.5.2",
    "next": "12.2.5",
    "react": "18.2.0",
    "react-dom": "18.2.0"
  },
  "devDependencies": {
    "common": "^0.2.5",
    "eslint": "^8.22.0",
    "eslint-config-google": "^0.14.0",
    "eslint-plugin-react": "^7.30.1"
  }
}
`

Can someone help me to solve it? Thanks

**Comment by icyJoseph** [comment]
Hi,

Got a repo to take a look at? What's that `common` package you've got installed? it is very much needed? what is it doing?
